src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Options/PackageSourcesPage.cs

```diff
@@ -237,6 +272,74 @@ private ExternalSettingOperationResult SetIsEnabledOnMachineWidePackageSources(
             return (result, hasAnyHiddenPropertyChanged);
         }
 
+        private (ExternalSettingOperationResult result, bool hasAnyHiddenPropertyChanged) SaveAuditSources(
+            IReadOnlyList<IDictionary<string, object>> auditSourceDictionaryList,
+            CancellationToken cancellationToken)
+        {
+            bool hasAnyHiddenPropertyChanged = false;
+            ExternalSettingOperationResult result;
+
+            try
+            {
+                List<PackageSource> auditSources = new List<PackageSource>(capacity: auditSourceDictionaryList.Count);
+                List<PackageSource> existingAuditSources = LoadAuditSources();
+                bool hasAnyPackageSourceNameChanged = false;
+
+                foreach (Dictionary<string, object> packageSourceDictionary in auditSourceDictionaryList)
+                {
+                    cancellationToken.ThrowIfCancellationRequested();
+
+                    string name = packageSourceDictionary[MonikerSourceName].ToString();
+                    string lookupName;
+
+                    // Package Sources that were pre-existing in the NuGet.Config when GetValueAsync was called will have a Package ID.
```

Comment:
```suggestion
                    // Package Sources that were pre-existing in the NuGet.Config when GetValueAsync was called will have a Source ID.
```

I assume that "Package ID" was used to be shorter than "Package Source ID".  But when I read "Package ID" I think of things like System.Text.Json, which doesn't make sense in the context of audit sources.

Reply:
Totally - not intentional, thanks